### ISSUE TITLE
Update to dart-sass?

### ISSUE THREAD

**Issue by willbuckingham** [Updater/node-sass-json-importer#97]
With the deprecation of LibSass / node-sass https://sass-lang.com/blog/libsass-is-deprecated , do you intend to change the peer dependency to sass?

Both have the same JS Api.  https://sass-lang.com/documentation/js-api

And they seem to work fine with this importer, however you get a terminal warning about unmet peer dependencies.  Honestly not sure if there's a way to make it an either/or peer dependency.  OR perhaps a rebrand of this library is coming down the line.

**Comment by brandonkramer** [comment]
Yes, would love to see this work with dart-sass without getting the terminal warning

**Comment by rootwork** [comment]
I've been using this with dart-sass regularly and it seems to be working fine, apart from the warning. Worth splitting into a node-sass release and a dart-sass release?

Or, alternatively, what about not having any peer dependencies defined, and then include a check somewhere to only warn if the user has _no_ version of Sass installed?
